static/app/views/detectors/components/details/error/index.tsx

```diff
@@ -64,12 +68,27 @@ function ResolveSection({project}: {project: Project}) {
 export function ErrorDetectorDetails({detector, project}: ErrorDetectorDetailsProps) {
   const organization = useOrganization();
 
+  const {selection} = usePageFilters();
+  const {start, end, period} = selection.datetime;
+  const timeProps =
+    start && end
+      ? {
+          start: getUtcDateString(start),
+          end: getUtcDateString(end),
+        }
+      : {
+          statsPeriod: period,
+        };
+
   return (
     <DetailLayout>
       <DetectorDetailsHeader detector={detector} project={project} />
       <DetailLayout.Body>
         <DetailLayout.Main>
-          <DetectorDetailsOngoingIssues />
+          <PageFiltersContainer>
```

Comment:
The page filters container should be rendered at the top level so that the page filters are hydrated before you access them, so in `detectors/detail.tsx`. That also means you don't have to add them for every detector type